Gosu/Constants.h

```diff
@@ -0,0 +1,248 @@
+#include <Gosu/Version.hpp>
+#include <Gosu/Buttons.hpp>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const char* Gosu_VERSION  = Gosu::VERSION.c_str();
+const char* Gosu_LICENSES = Gosu::LICENSES.c_str();
+
+unsigned Gosu_KB_RANGE_BEGIN     = Gosu::KB_RANGE_BEGIN;
```

Comment:
This should happen in a .cpp file, not in a .h file. Otherwise, if no .cpp file `#include`s this file, these symbols will not be defined, and if two .cpp files include this header, there will be linker errors about duplicate symbols.

Reply:
Moved this to a .cpp file.
`Gosu::VERSION` and `Gosu::LICENSES` are returning `null`, and I'm not sure why.